addon/components/time-series-chart.js

```diff
@@ -787,7 +784,9 @@ export default ChartComponent.extend(LegendMixin, TimeSeriesLabelerMixin, Floati
     'yScale',
     'xValueDisplayName',
     'yValueDisplayName',
-    'hasAxisTitles',
+    'hasAxisTitles', // backward compatability support.
```

Comment:
This is comment on the variable, not variable declaration. So I think we dont need jsdoc style